nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmITSupport.java

```diff
@@ -76,4 +106,40 @@ protected HelmClient helmClient(final URL repositoryUrl) throws Exception {
         repositoryUrl.toURI()
     );
   }
+
```

Comment:
Do we need 3 methods for creating helm client? 
`createHelmClient`, and two overloads for `helmClient`